frontend/tests/jest/tests/__snapshots__/ProductListItem.test.tsx.snap

```diff
@@ -8,10 +8,10 @@ exports[`ProductListItem renders and matches the snapshot 1`] = `
     role="group"
   >
     <div
-      class="css-mnl840"
+      class="css-11m6b36"
```

Comment:
I think this snapshot does not need to be updated?

Reply:
Branch was failing tests until I updated it. I thought it was odd too, but 🤷